serverless/internal/storage/fs/fs.go

```diff
@@ -53,14 +53,12 @@ type Storage struct {
 	// Note that nextSeq may be <= than the actual next available number, but
 	// never greater.
 	nextSeq uint64
-	// checkpoint is the latest known checkpoint of the log.
-	checkpoint log.Checkpoint
 }
 
 const leavesPendingPathFmt = "leaves/pending/%0x"
 
 // Load returns a Storage instance initialised from the filesystem.
-func Load(rootDir string, checkpoint *log.Checkpoint) (*Storage, error) {
+func Load(rootDir string, checkpoint log.Checkpoint) (*Storage, error) {
```

Comment:
This seems a bit messy to copy the struct in to only check the size. It's also the only reference left in this file to the `log` package, so maybe just pass in the uint64 directly?

Reply:
Yeah, I _um_'d and _ah_'d about that too.
I've changed it, ta.